client/src/components/Header/WordList/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import { baemin, baeminFont, greyBg1, greyLine, greySpan, lightBlack } from '@/static/style/common';
+
+type Histories = {
+  content: string;
+  day: string;
+  id: string;
+};
+
+type WordListProps = {
+  handleClick(e: React.MouseEvent<HTMLElement>): void;
+  histories: Array<Histories>;
+  nameForSearch: string;
+  recommendWords: string[];
+  idxForChoicedWord: number;
+};
+
+const SearchHistoryComponent = ({
+  handleClick,
+  histories,
+  nameForSearch,
+  recommendWords,
+  idxForChoicedWord,
+}: WordListProps) => {
+  const createWordList = () => {
+    if (nameForSearch.length === 0) {
+      return histories.length > 0 ? (
+        histories.map((history, idx) => (
+          <SearchHistory key={history.id}>
+            <Content id="content">{history.content}</Content>
+            <RightSide>
+              <Day>{history.day}</Day>
+              <Remove id="remove" className="fas fa-times" data-idx={idx}></Remove>
```

Comment:
제가 알기로는 i tag은 닫는 태그를 하나로 합쳐도 동작했던거 같은데 확인해 주실 수 있으실까요?
```suggestion
              <Remove id="remove" className="fas fa-times" data-idx={idx}/>
```

Reply:
아 i 상관없습니다 다만 자동완성이 저렇게 되다보니 일일히 고쳐주기가 은근... ㅋㅋ